### PR TITLE
Login signup

### PR DESCRIPTION
**Pull request by saifudinbadani**
Please review login & signup pages:

- no functionality added
- review the structure of form and file.

Screen Shot 2022-03-23 at 3 29 18 pm
Screen Shot 2022-03-23 at 3 29 24 pm

### CODE REVIEW

**Review by samyakshah3008 — APPROVED** [review]
Good naming conventions 
Clean code 
It was easy for reviewing due to small PR 

Good for approval ✅

**Comment by samyakshah3008** [comment]
Make sure you resolve merge conflicts

File: src/index.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
-import { HomePage, ProductListing  } from "./pages/index";
+import { HomePage, ProductListing, Login, Signup  } from "./pages/index";
```
Comment (MurtazaBagwala07) [inline src/index.js:5]: it will work even if you write is like "./pages" .ie index is not necessary

**Review by MurtazaBagwala07 — APPROVED** [review]
Folder structure is easy to understand
Code is clean 
Good job

**Pull request merged**